### PR TITLE
[DOCS] Endpoint Management docs: consistency fixes, prerequisite callout

### LINKED ISSUE #1826: [DOCS] Endpoint Management docs: consistency fixes, prerequisite callout
## Description 

Most of the pages for endpoint management (the **Manage** section in the side nav) are very similar, so they should have consistent docs. Over time, phrasing has been edited on a page-by-page or feature-by-feature basis, without maintaining consistency across all features.

Additionally, the @elastic/security-docs team discussed using a dedicated "Prerequisites" callout for noting user role requirements, so while we're taking a more global look at these pages, we can also add this box across all Manage pages.

Sample syntax:
```
.Title
*****
Text text 

* Bullet
* Bullet

Text text text 

*****
```
## Acceptance Criteria

- [ ] Add Prerequisites callout box
- [x] Rename "actions button" to "actions menu"
- [x] Various tweaks to "View and manage _x_" sections (Probably use Blocklist as a model, since it's the most recently worked-on page)

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves #1826.

Previews:
- [Trusted applications | View and manage...
](https://security-docs_1930.docs-preview.app.elstc.co/guide/en/security/master/trusted-apps-ov.html#trusted-apps-list)
- [Event filters | View and manage...
](https://security-docs_1930.docs-preview.app.elstc.co/guide/en/security/master/event-filters.html#manage-event-filters)
- [Host isolation exceptions | View and manage...
](https://security-docs_1930.docs-preview.app.elstc.co/guide/en/security/master/host-isolation-exceptions.html#manage-host-isolation-exceptions)
- [Blocklist | View and manage...](https://security-docs_1930.docs-preview.app.elstc.co/guide/en/security/master/blocklist.html#manage-blocklist)

Incidental updates (other instances of "Actions button" renamed to "Actions menu"):
- [Rule exceptions and value lists | Add exceptions to a rule](https://security-docs_1930.docs-preview.app.elstc.co/guide/en/security/master/detections-ui-exceptions.html#detection-rule-exceptions)
- [Rule exceptions and value lists | Add Elastic Endpoint Security exceptions](https://security-docs_1930.docs-preview.app.elstc.co/guide/en/security/master/detections-ui-exceptions.html#endpoint-rule-exceptions)
- [Monitor and troubleshoot rule executions | Troubleshoot gaps](https://security-docs_1930.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-monitor.html#troubleshoot-gaps)
- [Troubleshooting | Management tools](https://security-docs_1930.docs-preview.app.elstc.co/guide/en/security/master/ts-management.html)

### CODE REVIEW

**Review by nastasha-solomon — APPROVED** [review]

File: docs/management/admin/host-isolation-exceptions.asciidoc [hunk 1]
```diff
@@ -53,7 +53,7 @@ You can delete a host isolation exception, which removes it entirely from all {e
 
 To delete a host isolation exception:
 
-. Click the actions button (**...**) for the exception you want to delete, then select **Delete Exception**.
-. On the dialog that opens, verify that you are removing the correct host isolation exception, then click **Remove exception**. A confirmation message is displayed.
+. Click the actions menu (**...**) for the exception you want to delete, then select **Delete Exception**.
```
Comment (jmikell821) [inline docs/management/admin/host-isolation-exceptions.asciidoc:56]:
```suggestion
. Click the actions menu (**...**) on the exception you want to delete, then select **Delete Exception**.
```

File: docs/management/admin/trusted-apps.asciidoc [hunk 2]
```diff
@@ -51,19 +51,19 @@ NOTE: You can also select the `Per Policy` option without immediately assigning
...
 [discrete]
 [[edit-trusted-app]]
 === Edit a trusted application
-You can individually configure each trusted application. With a Platinum or Enterprise subscription, you can also change the policies applied to a trusted application.
+You can individually modify each trusted application. With a Platinum or Enterprise subscription, you can also change the policies that a trusted application is assigned to.
 
 To edit a trusted application:
 
-. Click the actions button (*...*​) for the trusted application you want to edit, then select *Edit trusted application*.
+. Click the actions menu (*...*) for the trusted application you want to edit, then select *Edit trusted application*.
```
Comment (jmikell821) [inline docs/management/admin/trusted-apps.asciidoc:66]:
```suggestion
. Click the actions menu (*...*) on the trusted application you want to edit, then select *Edit trusted application*.
```

File: docs/management/admin/trusted-apps.asciidoc [hunk 3]
```diff
@@ -74,5 +74,5 @@ You can delete a trusted application, which removes it entirely from all {endpoi
 
 To delete a trusted application:
 
-. Click the actions button (*...*) for the trusted application you want to delete, then select *Delete trusted application*.
+. Click the actions menu (*...*) for the trusted application you want to delete, then select *Delete trusted application*.
```
Comment (jmikell821) [inline docs/management/admin/trusted-apps.asciidoc:77]:
```suggestion
. Click the actions menu (*...*) on the trusted application you want to delete, then select *Delete trusted application*.
```

**Review by jmikell821 — COMMENTED** [review]
Just a few suggestions to use `on` vs. `for`.  LGTM - thanks for these updates!

**Review by benironside — APPROVED** [review]

**Pull request merged**